t/ui/16-tests_job_next_previous.t

```diff
@@ -276,5 +276,38 @@ subtest 'bug reference shown' => sub {
     );
 };
 
+subtest 'server-side limit has precedence over user-specified limit' => sub {
+    my $limits = OpenQA::App->singleton->config->{misc_limits};
+    $limits->{next_ajax_max_limit} = 5;    # set maximum limit for next
+    $limits->{previous_ajax_max_limit} = 5;    # set maximum limit for previous
+    $limits->{next_ajax_default_limit} = 2;    # set default limit for next
+    $limits->{previous_ajax_default_limit} = 2;    # set default limit for previous
```

Comment:
IMHO the comments are redundant and should be removed. If the variable names are not clear enough then the variables should be renamed rather than explained by comments

Reply:
ok, i removed the comments and changed the naming of the variables 
(replaced 'ajax' with 'jobs' e.g. next_ajax_max_limit -> next_jobs_max_limit, as Marius had suggested)